core/src/components/picker-column/test/disabled/picker-column.e2e.ts

```diff
@@ -105,9 +68,8 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
 /**
  * This behavior does not vary across directions.
  */
-// TODO FW-5580 fix this
 configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
-  test.describe.skip(title('picker-column: disabled column rendering'), () => {
+  test.describe(title('picker-column: disabled column rendering'), () => {
```

Comment:
I forgot to unskip these tests when working on previous PRs. However, I ran them all locally and verified that they passed during development.